src/snowflake/cli/api/commands/snow_typer.py

```diff
@@ -79,6 +79,8 @@ def pre_execute():
     @staticmethod
     def process_result(result):
         """Command result processor"""
+        if not result:
+            return
```

Comment:
Because we have logs commands that currently do "streaming" of results

Reply:
nit: Worth a comment in the code then?

Reply:
Added